forc/GFS/README.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT variables : from the output of GFSv16,
+    ==> longitude, latitude, time, DLWRF_surface, ULWRF_surface, DSWRF_surface, UFLX_surface, VFLX_surface, SHTFL_surface,
+      LHTFL_surface, PRATE_surface, UGRD_10maboveground, VGRD_10maboveground, PRES_surface, TMP_1hybridlevel, 
+      SPFH_1hybridlevel, UGRD_1hybridlevel, VGRD_1hybridlevel, SPFH_2maboveground, TMP_2maboveground, LAND_surface, 
+      delz, PRATE_surface, pres_hyblev1, precp, fprecp, ICEC_surface
+
+* ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
```

Comment:
This file is the input, but it is not the input to the fortran code.
There is conversion from grib2 to netCDF using wgrib2, and then concatenation with other netcdf files, that are finally producing the input file.
None of that is clear from looking at the scripts or the code.

Reply:
- The whole inputs are listed in the line 22 - 80 of the code of conv_gfs2datm_long_beta4.f.
- Some comments of the process were added in conv_gfs2datm.fort_beta4.csh and get_gfs_beta4_gdas.csh for the conversion of grib2 to nc and concatenation of nc files.